src/sunspecdemo/__init__.py

```diff
@@ -0,0 +1,2 @@
+#0.0.0 dev build placeholder
```

Comment:
```suggestion
# For development, the __version__ is set to the build placeholder 0.0.0
# For release, the __version__ is the actual released version
```

The comment is sort of confusing when viewed from the perspective of the release package. I suggest this more detailed comment instead.